test/api/v3/integration/groups/POST-groups_groupId_join.test.js

```diff
@@ -16,7 +16,39 @@ describe('POST /group/:groupId/join', () => {
     });
   });
 
-  context('Accepting invitation to a private guild', () => {
+  context('Joining a public guild', () => {
+    let user, joiningUser, publicGuild;
+
+    beforeEach(async () => {
+      let {group, groupLeader} = await createAndPopulateGroup({
+        groupDetails: {
+          name: 'Test Guild',
+          type: 'guild',
+          privacy: 'public',
+        },
+      });
+
+      publicGuild = group;
+      user = groupLeader;
+      joiningUser = await generateUser();
+    });
+
+    it('allows non-invited users to join public guilds', async () => {
```

Comment:
I guess that's a general question, @crookedneighbor @paglias  should I have duplicate tests for everything that is both true for private and public guilds, or only where there is a difference in the code that changes the behavior depending on the guild type?